identity-service/src/notifications/sendNotifications/index.js

```diff
@@ -22,6 +22,19 @@ function getUserIdsToNotify (notifications) {
       case notificationTypes.MilestoneListen:
       case notificationTypes.TierChange:
         return userIds.concat(notification.initiator)
+      case notificationTypes.Tip:
+        // For the case of a tip, need to add both senderId and receiverId
+        const receiverId = notification.initiator
+        const senderId = notification.metadata.entity_id
+        return userIds.concat([receiverId, senderId])
+      case notificationTypes.Reaction:
+        // For reactions, add the tip_sender_id in the reacted_to_entity
+        return userIds.concat(notification.metadata.reacted_to_entity.tip_sender_id)
```

Comment:
I know there's no other reactions yet, but this feels ripe to bite us when we add new types. Are we sure we don't want to check the reaction type first?

Reply:
Am down to add it